google/auth/aws.py

```diff
@@ -338,211 +338,141 @@ def _generate_authentication_header_map(
     return authentication_header
 
 
-class Credentials(external_account.Credentials):
-    """AWS external account credentials.
-    This is used to exchange serialized AWS signature v4 signed requests to
-    AWS STS GetCallerIdentity service for Google access tokens.
-    """
+class AwsSecurityCredentials(object):
+    """An object that models AWS security credentials with an optional session token."""
```

Comment:
This seems like a good candidate for a Dataclass since this codebase now uses at a minimum Python 3.7.

Reply:
done.